"X509_load_cert_crl_file" HTTPS error in packaged deis CLI

Trying to communicate with an HTTPS-fronted Deis controller from the `deis` v1.1.0 PyInstaller-created client leads to an error that does not happen running `deis` locally, and presumably not with the `pip install`-ed version either:

``` console
$ deis register https://deis.deis.rocks/ --username=matt --password=testing --email=matt@deis.rocks
Couldn't connect to the Deis Controller:
[Errno 185090050] _ssl.c:343: error:0B084002:x509 certificate routines:X509_load_cert_crl_file:system lib
Make sure that the Controller URI is correct and the server is running.
$ ./client/deis.py register https://deis.deis.rocks/ --username=matt --password=testing --email=matt@deis.rocks
Registration failed: <html>
<head><title>404 Not Found</title></head>
```

[This PyInstaller issue](https://github.com/kennethreitz/requests/issues/557#issuecomment-51262733) seems very relevant.

--- a/client/Makefile
+++ b/client/Makefile
@@ -1,6 +1,6 @@
 
 build: setup-venv
-	venv/bin/pip install docopt==0.6.2 python-dateutil==2.2 PyYAML==3.11 requests==2.4.3 pyinstaller==2.1 termcolor==1.1.0
+	venv/bin/pip install docopt==0.6.2 python-dateutil==2.2 PyYAML==3.11 requests==2.4.3 git+https://github.com/pyinstaller/pyinstaller@7413317 termcolor==1.1.0
 	venv/bin/pyinstaller deis.spec
 	chmod +x dist/deis
 

--- a/controller/dev_requirements.txt
+++ b/controller/dev_requirements.txt
@@ -5,7 +5,7 @@ requests==2.4.3
 termcolor==1.1.0
 
 # PyInstaller builds client binaries
-PyInstaller==2.1
+git+https://github.com/pyinstaller/pyinstaller@7413317
 
 # Deis documentation requirements
 Sphinx>=1.2.3

--- a/tests/bin/build-deis-cli.sh
+++ b/tests/bin/build-deis-cli.sh
@@ -4,5 +4,5 @@
 
 virtualenv --system-site-packages venv
 . venv/bin/activate
-pip install docopt==0.6.2 python-dateutil==2.2 PyYAML==3.11 requests==2.4.3 pyinstaller==2.1 termcolor==1.1.0
+pip install docopt==0.6.2 python-dateutil==2.2 PyYAML==3.11 requests==2.4.3 git+https://github.com/pyinstaller/pyinstaller@7413317 termcolor==1.1.0
 make -C client/ client

